packages/cli/src/permissions/roles.ts

```diff
@@ -61,12 +25,47 @@ export const ownerPermissions: Scope[] = [
 	'eventBusDestination:delete',
 	'eventBusDestination:list',
 	'eventBusDestination:test',
+	'externalSecretsProvider:create',
+	'externalSecretsProvider:read',
+	'externalSecretsProvider:update',
+	'externalSecretsProvider:delete',
+	'externalSecretsProvider:list',
+	'externalSecretsProvider:sync',
+	'externalSecret:list',
+	'ldap:manage',
+	'ldap:sync',
+	'logStreaming:manage',
+	'orchestration:read',
+	'orchestration:list',
+	'saml:manage',
+	'sourceControl:pull',
+	'sourceControl:push',
+	'sourceControl:manage',
+	'tag:create',
+	'tag:read',
+	'tag:update',
+	'tag:delete',
+	'tag:list',
+	'user:create',
+	'user:read',
+	'user:update',
+	'user:delete',
+	'user:list',
+	'user:resetPassword',
+	'variable:create',
+	'variable:read',
+	'variable:update',
+	'variable:delete',
+	'variable:list',
+	'workflow:create',
+	'workflow:read',
+	'workflow:update',
+	'workflow:delete',
+	'workflow:list',
+	'workflow:share',
 ];
 export const adminPermissions: Scope[] = ownerPermissions.concat();
 export const memberPermissions: Scope[] = [
-	'user:list',
-	'variable:list',
-	'variable:read',
```

Comment:
These are required on the backend for the user to be able to get the user list for sharing and to get the variables they can use.

Reply:
My mistake! Got messed up from the conflicts.